doc/src/manual/performance-tips.md

```diff
@@ -81,7 +81,9 @@ problem with type-stability. Consequently, in addition to the allocation itself,
 that the code generated for your function is far from optimal. Take such indications seriously
 and follow the advice below.
 
-As a teaser, note that an improved version of this function allocates no memory (except to pass
+For more serious benchmarking, consider the [`BenchmarkTools.jl`](https://github.com/JuliaCI/BenchmarkTools.jl) package which evaluates the function multiple times in order to reduce noise.
+
+As a teaser, an improved version of this function allocates no memory (except to pass
```

Comment:
> As a teaser, an improved version of this function allocates no memory (except to pass back the result back to the REPL)

What does this even mean? Why would pass the result back to the REPL allocate memory?